File: scratch/analyze_css.py
import re

with open('styles/globals.css', 'r', encoding='utf-8') as f:
    content = f.read()

# Find all blocks and their selectors
blocks = re.findall(r'([^{]+)\{([^}]+)\}', content)

for selector, body in blocks:
    selector = selector.strip()
    body = body.strip()
    if 'mesa' in selector:
        print(f"Selector: {selector}")
        for line in body.split('\n'):
            print(f"  {line.strip()}")
        print("-" * 40)

File: scratch/analyze_css.js
const fs = require('fs');

const content = fs.readFileSync('styles/globals.css', 'utf-8');

// Match css rules
const regex = /([^{]+)\{([^}]+)\}/g;
let match;
while ((match = regex.exec(content)) !== null) {
  const selector = match[1].trim();
  const body = match[2].trim();
  if (selector.includes('mesa')) {
    console.log(`Selector: ${selector}`);
    body.split('\n').forEach(line => {
      console.log(`  ${line.trim()}`);
    });
    console.log("-".repeat(40));
  }
}
